redis/README.md

```diff
@@ -0,0 +1,43 @@
+## Redis Monitoring Template
+
+This InfluxDB Template can be used to monitor Redis.
+
+![Redis Dashboard Screenshot](img/redis_dashboard.png)
+
+### Included Resources
+
+- 1 Bucket: `redis`, 7d retention
+- Labels: Telegraf Plugin Labels
+- 1 Telegraf Configuration
+- 1 Dashboard: `Redis`
+- 1 Variable: `bucket`
+
+## Setup Instructions
+    
+  The data for the dashboard is populated by the included Telegraf configuration. The Telegraf Configuration requires the following environment variables
+    
+  - `INFLUX_TOKEN` - The token with the permissions to read Telegraf configs and write data to the `telegraf` bucket. You can just use your master token to get started.
+  - `INFLUX_ORG` - The name of your Organization (this will be your email address on the InfluxDB Cloud free tier)
+  - `INFLUX_HOST` - The URL of your InfluxDB host (this can your localhost, a remote instance, or InfluxDB Cloud)
+
+  You **MUST** set these environment variables before running Telegraf using something similar to the following commands
+    
+  - This can be found on the `Load Data` > `Tokens` page in your browser: `export INFLUX_TOKEN=TOKEN`
+  - Your Organization name can be found on the Settings page in your browser: `export INFLUX_ORG=my_org`
+
+## Running Telegraf
```

Comment:
It looks like this has to be run on the same host as Redis, since `servers = ["tcp://localhost:6379"]` is set in the configuration. Can we make this a variable or be explicit in the instructions that it must be run on the same host?